site/reference/cli/fn/eval/README.md

```diff
@@ -125,11 +125,21 @@ fn-args:
   3. OUT_DIR_PATH: output resources are written to provided directory.
      The provided directory must not already exist.
 
+--type, t;
+  Specify the function type. Accept value `mutator` (default), `validator`. 
+  If used with `--save`, this flag will save the evaluated function to the corresponding
+   Kptfile section: `.pipeline.mutators` if type is `mutator`; `.pipeline.validators` if type
+    is `validator`.
+
 --results-dir:
   Path to a directory to write structured results. Directory will be created if
   it doesn't exist. Structured results emitted by the functions are aggregated and saved
   to `results.yaml` file in the specified directory.
   If not specified, no result files are written to the local filesystem.
+  
+--save, s:
+  Save the function image and fn-config to Kptfile. Require ` + "`" + `--image` + "`" + `.
+  
```

Comment:
You can separate the docs change in a separate PR and merge them after the kpt release.

This is because docs change will go live on the site immediately once the PR is merged but kpt's latest release won't have the changes.

Reply:
oh! Now I recalled why we removed the --save docs. kk.

Reply:
Separate the docs in another PR https://github.com/GoogleContainerTools/kpt/pull/2977